build.gradle

```diff
@@ -63,6 +63,9 @@ jar {
                 '-exportcontents': 'org.dataloader.*',
                 '-removeheaders': 'Private-Package')
     }
+    bnd('''
+Import-Package: org.jspecify.annotations;resolution:=optional,*
+''')
```

Comment:
Small nitpick, your title mentions org.jspecify.* but this PR is only excluding `annotations`. In the corresponding GraphQL Java PR, `org.specify.*` is set as optional.

I've merged in this PR. Thanks for the PR!